crypto/src/test_utils/zero_crypto.rs

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * This file contains an implementation of a negligible-cost (apart from some ocassional pk copying) trivial
+ * cryptographic scheme. The purpose of this library is to allow seamless benchmarking of systems without
+ * taking into account the cost of cryptographic primitives - and hence providing a theoretical maximal
+ * throughput that a system could achieve if the cost of crypto is optimized away.
+ */
+use base64ct::{Base64, Encoding};
+use digest::Digest;
+use rand::Rng;
+use std::{
+    fmt::{self, Display},
+    str::FromStr,
+};
+
+use crate::{pubkey_bytes::PublicKeyBytes, serde_helpers::keypair_decode_base64};
+use serde::{
+    de::{self},
+    Deserialize, Serialize,
+};
+use serde_with::serde_as;
+
+use signature::{Signature, Signer, Verifier};
+
+use crate::traits::{
+    AggregateAuthenticator, Authenticator, EncodeDecodeBase64, KeyPair, SigningKey, ToFromBytes,
+    VerifyingKey,
+};
+use sha3::Sha3_256;
+
+///
+/// Define Structs
+///
+
+const PRIVATE_KEY_LENGTH: usize = 32;
```

Comment:
We can use smaller keys sizes here to reduce the cost impact even further. I think the minimum we could go and ensure no collisions is 10-16 bytes for both keys. I'd pick 16.

Reply:
I've kept private key to 32 bytes, as this is just helpful for deriving it from seed.